astropy/io/fits/tests/test_image.py

```diff
@@ -1010,7 +1010,9 @@ def test_comp_image_hcompress_image_stack(self):
         hdu.writeto(self.temp('test.fits'))
 
         with fits.open(self.temp('test.fits')) as hdul:
-            assert (hdul['SCI'].data == cube).all()
+            # HCOMPRESSed images are allowed to deviate from the original by
+            # about 1/quantize_level of the RMS in each tile. 
+            assert np.abs(hdul['SCI'].data - cube).max() < 1./15.
```

Comment:
@olebole - I'm not too familiar with this, but isn't the quantize level 16 not 15, and does this assume the RMS is 1?